eval.go

```diff
@@ -202,9 +187,14 @@ func (e *CallExpr) eval(app *App, args []string) {
 			}
```

Comment:
@KenjiTakahashi Sorry, I'm failing to see where you mean. Can you describe the place somehow? Under `case "open"` somewhere?